src/dashboard/pages/dashboard.py

```diff
@@ -0,0 +1,74 @@
+from collections import OrderedDict
+
+import dash
+from dash import dcc, html
+import plotly.express as px
+
+from dashboard.components.icon import icon
+
+dash.register_page(__name__, path_template="/dashboard/<dashboard_id>")
+
+# Ordered dict containing the name of the button and its corresponding icon name
+# Find the icon names here https://fonts.google.com/icons?icon.set=Material+Symbols
+buttons = OrderedDict(
```

Comment:
* Module level constants should follow `SCREAMING_SNAKE_CASE`.
* No need to use ordered dict here, since dict is orderered by default[1]. I'm not even sure a dict is appropriate, since it is never indexed.

[1] https://mail.python.org/pipermail/python-dev/2017-December/151283.html